grouper/service_account.py

```diff
@@ -10,7 +10,19 @@ class ServiceAccountNotFound(Exception):
 
 
 def is_service_account(session, user=None, group=None):
-
+    # Type: Session, User, Group -> bool
```

Comment:
Should be `type` per [PEP 484](https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code).